lib/matplotlib/mlab.py

```diff
@@ -3853,9 +3846,10 @@ def poly_between(x, ylower, yupper):
     Return value is *x*, *y* arrays for use with
     :meth:`matplotlib.axes.Axes.fill`.
     """
-    if (ma.isMaskedArray(ylower) or ma.isMaskedArray(yupper) or
-            ma.isMaskedArray(x)):
-        numpy = ma
+    if (isinstance(ylower, np.ma.MaskedArray) or
+            isinstance(yupper, np.ma.MaskedArray) or
+            isinstance(x, np.ma.MaskedArray)):
```

Comment:
This is getting pretty clunky.  I'm almost ready to argue for standardizing on `isMA` as a readability win.  Also, there is nothing wrong with making code a bit more concise and readable by importing frequently used symbols at the top so as to avoid so much `np.ma.` repetition in a case like this.